probe-rs/src/architecture/arm/core/armv7a.rs

```diff
@@ -321,6 +325,30 @@ impl<'probe> CoreInterface for Armv7a<'probe> {
         Ok(dbgdscr.halted())
     }
 
+    fn status(&mut self) -> Result<crate::core::CoreStatus, Error> {
```

Comment:
Why is this here? :)

Reply:
The github diff incorrectly reports this as new, because I sorted the methods to be in the same order as in the `CoreInterFace` trait definition. Just to make it easier to maintain.